azure-keyvault-cryptography/src/main/java/com/microsoft/azure/keyvault/cryptography/algorithms/Rs256.java

```diff
@@ -46,7 +46,7 @@
 			}
 			
 			// Construct the encoded message
-			byte[] EM = EMSA_PKCS1_V1_5_ENCODE_HASH(digest, _emLen, "SHA-256");
+			byte[] EM = EMSA_PKCS1_V1_5_ENCODE(digest, _emLen, "SHA-256");
```

Comment:
>byte[] EM = EMSA_PKCS1_V1_5_ENCODE(digest, _emLen, "SHA-256"); [](start = 3, length = 62)

This needs to stay ENCODE_HASH this methods parameter 'digest' implies that it takes a prehashed digest.  You'll need to update your calls to prehash the digest.